src/include/storage/copier/struct_column_chunk.h

```diff
@@ -36,6 +36,7 @@ class StructColumnChunk : public ColumnChunk {
     void write(const common::Value& val, uint64_t posToWrite) final;
     void copyStructFromArrowStruct(
         arrow::Array* array, common::offset_t startPosInChunk, uint32_t numValuesToAppend);
+    template<typename T>
```

Comment:
Replace `T` with `ARROW_TYPE` is more clear what type this function is switching on.